examples/cron-workflow-multiple-schedules.yaml

```
@@ -0,0 +1,22 @@
+apiVersion: argoproj.io/v1alpha1
+kind: CronWorkflow
+metadata:
+  name: hello-world-multiple-schedules
+spec:
+  schedules: 
+  - "*/3 * * * *"
+  - "*/2 * * * *"
```

```suggestion
  schedules: # v3.6 and after
    - "*/3 * * * *"
    - "*/2 * * * *"
```

- add in-line version reference here
- consistent indentation